parlai/tasks/dealnodeal/agents.py

```diff
@@ -64,6 +64,9 @@ def __init__(self, opt, shared=None):
 
         self.reset()
 
+    def __len__(self):
+        return len(self.episodes)
+
```

Comment:
is len episodes or examples?

Reply:
Because each episode has one example, the len for this teacher is len episodes